src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -6,9 +6,31 @@
 
 public class RegistrationServiceImpl implements RegistrationService {
     private final StorageDao storageDao = new StorageDaoImpl();
+    private static final int MIN_AGE = 18;
+    private static final int MIN_LENGTH_DATA = 6;
 
     @Override
     public User register(User user) {
-        return null;
+        if (checkLoginIfPresent(user) != null ) {
```

Comment:
This condition checks if the user with the given login already exists, but according to the checklist, we should not complicate if-else statements. Consider splitting the conditions into separate if statements without 'else' for better readability and maintainability.